clients/admin-ui/src/features/datastore-connections/ConnectionGridItem.tsx

```diff
@@ -112,9 +80,7 @@ const ConnectionGridItem: React.FC<ConnectionGridItemProps> = ({
         />
       </Flex>
       <Text color="gray.600" fontSize="sm" fontWeight="sm" lineHeight="20px">
-        {getConnectorDisplayName(
-          connectionData.connection_type as ConnectionType
-        )}
+        {connectionData.description ? connectionData.description : <br />}
```

Comment:
Does it make more sense to give the element a margin rather than enter a `<br />` element here, and instead do `: null` ? I think Text acts like a block item, right?

Reply:
<br/> acts as a placeholder when there is no description. Discussed this with Simon and Adam (UI/UX).